### DISCUSSION TITLE
Windows daily builds are 22 bytes long since at least Oct 13

### DISCUSSION THREAD

**Discussion by anttin2020** [sqlitebrowser/sqlitebrowser#3487] [Q&A]
Hi!

I have version 3.12.99 (Oct 1) but when I now try to go to daily builds, both Windows versions are 22 bytes long. They still look and act like ZIPs but they are empty. That has been the case at least from Fri Oct 13 as you can see here:
https://nightlies.sqlitebrowser.org/win64/ and of course in latest/ which probably is just symlink.

Please fix this soon.

Thanks!

**Comment by justinclift** [comment]
Whoops.  Thanks for pointing this out.  Looking into it now. :smile:

**Reply by justinclift** [reply]
While I'm at it, the SQLCipher builds turned out to be using a slightly older version (`4.5.4` vs `4.5.5`) as well.

I'll update that while fixing this, as long as it doesn't break anything. :smile:

**Comment by justinclift** [comment]
Heh Heh Heh.  Looks like we'll need to update the MS Visual Studio license in a few days as well:

![image](https://github.com/sqlitebrowser/sqlitebrowser/assets/406299/e188580f-22d1-47da-89bc-343509d05d7d)

Hopefully that works out ok too.

**Comment by justinclift** [comment]
Looks like the problem is in the fileio extension:

![image](https://github.com/sqlitebrowser/sqlitebrowser/assets/406299/a6a04b93-4153-4d2a-862d-05576ade2a45)

and:

![image](https://github.com/sqlitebrowser/sqlitebrowser/assets/406299/bb227014-6d9c-483d-a642-3675ddbce7eb)

I'll disable the building and inclusion of that extension for now, which should at least get the builds working again.

**Reply by anttin2020** [reply]
No hurry, the 3.12.99 is working fine, I just wanted to notify that automated builds are not working and for other people it may be bad as that working version is not latest anymore, still available in win64 folder though :)

**Reply by justinclift** [reply]
@anttin2020 There's new Win64 build (msi and zip) in the `latest` directory.  Do you have time to try it out and make sure things look ok from your point of view?

I don't personally run windows, and testing the software in the build vm itself isn't a great approach as it won't show up dependency problems in end user installs (etc).

**Reply by anttin2020** [reply]
Yes it's working fine, thank you very much and thank you for this great tool :)

**Reply by justinclift** [reply]
Awesome.  You're welcome, and thanks for letting us know about the problem. :smile:

**Comment by justinclift** [comment]
Just recreated the Win64 builds again, and also generated the Win32 builds for tday as well.  I'll close this issue now, as we should be good for now.  At least until the MSVC license expires in 8 days. :wink:

**Comment by anttin2020** [comment]
Broken again... 22 bytes

**Reply by justinclift** [reply]
Thanks for letting us know.  I'll have to take a look late tomorrow, as I really need to get some sleep right now.

**Reply by anttin2020** [reply]
Oh you noticed this even this is closed, I also created new discussion, sorry :)
Please remove it, I could not close it.

No hurry, sleep is very important, just wanted to tell that this weird problem appeared again :)

**Reply by justinclift** [reply]
Found the problem.  It was the Microsoft Visual Studio 2017 license expiring.  I've just managed to renew it (which I doubt will work again), so we should probably figure out a new Windows build process before the next expiry.

Windows nightly builds are currently happening now.

---

The problem with the license activation this time was that the "sign in" page for the microsoft account was completely blank.  As in window appears with completely white contents.  Luckily it had a text cursor icon in part of it, so I was able to fill in the username and lucky-guess the right number of presses on the tab key + enter key to get to the (working) password entry screen.

Should have taken a screenshot, but didn't think of it until afterwards. :frowning:

**Reply by justinclift** [reply]
@anttin2020 Newly created Windows nightly builds are online again. :smile:

**Reply by anttin2020** [reply]
@justinclift  Oh that was interesting problem, good that you managed to work around it :o 
Thank you very much, yes now it works again :)
Have a nice weekend!
